modules/ob-installing-buildah-strategy.adoc

```diff
@@ -0,0 +1,22 @@
+// This module is included in the following assembly:
+//
+// builds/installing-sample-build-strategies.adoc
+
+:_content-type: PROCEDURE
+[id="installing-buildah-strategy_{context}"]
+= Installing buildah strategy
+
+You can install the `buildah` cluster build strategy, which does not require a daemon to run and is available for use by unprivileged users.
+
+.Prerequisites
+
+* You have installed the `oc` CLI.
+
+.Procedure
+
+* Run the following command to install the `buildah` strategy:
```

Comment:
Same comment as above: " To install the `buildah` strategy, run the following command:"